packages/css/src/components/form-error-list/form-error-list.scss

```diff
@@ -0,0 +1,4 @@
+.ams-form-error-list {
+  // The offset is doubled to compensate for the Alert border
+  outline-offset: var(--ams-form-error-list-outline-offset);
```

Comment:
We do [the same for File Input](https://github.com/Amsterdam/design-system/blob/develop/packages/css/src/components/file-input/file-input.scss#L24) but without an extra token. Which of the two approaches do you prefer?